doc/install.rst

```diff
@@ -4,79 +4,98 @@
 Installing scikit-learn
 =======================
 
-.. note::
+There are different ways to get scikit-learn installed:
```

Comment:
I would rather keep this main section focused on installing the stable release. But I agree we can refactor the instructions to build scikit-learn from source by OS in another PR.

Reply:
@jeremiedbb @ogrisel I have finally moved "nightly builds" in `install.rst` ... but I'm not sure this was the conclusion of our discussion... so, sorry if it was'nt...

Reply:
I think the conclusion was to keep it :) 
making this page about getting the development version of scikit-learn (instead of the vagueish "advanced installation")

Reply:
Arrrgh... :) fixed